projects/angular/src/layout/vertical-nav/vertical-nav.spec.ts

```diff
@@ -482,6 +484,23 @@ export default function (): void {
         fixture.destroy();
       });
 
+      it('supports an input to change aria-label on collapsible vertical nav trigger', () => {
+        fixture.componentInstance.collapsible = true;
+        fixture.detectChanges();
+
+        expect(vertNavService.collapsible).toBe(true);
```

Comment:
```suggestion
        expect(vertNavService.collapsible).toBe(true);
```

This assertion isn't really needed in this test. This is tested in "supports an input to enable the collapsible behavior of the nav", so this is redundant.

Reply:
Corrected in commitr 9bd34d7.